Configuration/TCA/Overrides/sys_file_reference.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
+
+$additionalColumns = [
+    'language' => [
+        'label' => 'LLL:EXT:core/Resources/Private/Language/locallang_general.xlf:LGL.language',
+        'description' => 'LLL:EXT:html5mediakit/Resources/Private/Language/locallang_csh_media.xlf:language.description',
+        'config' => [
+            'type' => 'input',
```

Comment:
One idea for the future (as soon as static_info_tables is compatible with TYPO3 12):

We could add a check if the static_info_tables Extension is loaded and if it is the case we could replace the plain input field with a language select of the static_info_tables Extension, see:

https://docs.typo3.org/typo3cms/extensions/static_info_tables/stable/Manual/Index.html#configuration-of-backend-forms

But this is only an idea for the future and no TODO for this PR.